### PR TITLE
CQL Query logging changes.

### PR DESCRIPTION
**Pull request by kathirsvn**
**What this PR does**:
Enables CQL Query logging for slow and errored queries.

**Checklist**
- [X] Changes manually tested
- [X] Automated Tests added/updated
- [X] Documentation added/updated
- [X] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -28,6 +28,22 @@ datastax-java-driver {
...
+    classes = [RequestLogger]
+    logs {
+      success.enabled = false
+      slow {
+        threshold = 1 second
+        enabled = true
+      }
+      error.enabled = true
+      max-query-length = 500
+      show-values = true
```
Comment (jeffreyscarpenter) [inline src/main/resources/application.conf:41]: I think we should set this to false so that we don't get any customer data in the logs

**Review by jeffreyscarpenter — CHANGES_REQUESTED** [review]

File: src/main/resources/application.conf [hunk 2]
```diff
@@ -28,6 +28,22 @@ datastax-java-driver {
     }
   }
   basic.request.timeout = 10 seconds
+  advanced.request-tracker {
```
Comment (tatu-at-datastax) [inline src/main/resources/application.conf:31]: nit: should come before `basic.request.timeout` to keep things in alphabetic order.

File: src/main/resources/application.conf (hunk 1 above)
Comment (tatu-at-datastax) [reply src/main/resources/application.conf:41]: That does seem safer default choice, +1.

Comment (kathirsvn) [reply src/main/resources/application.conf]: Agreed. Changed to `false` in [this commit](https://github.com/stargate/jsonapi/pull/754/commits/b70a3266bab05de5f9c68f73bd0963e3a3e399f2).

File: src/main/resources/application.conf (hunk 2 above)
Comment (kathirsvn) [reply src/main/resources/application.conf:30]: Changed in [this commit](https://github.com/stargate/jsonapi/pull/754/commits/b70a3266bab05de5f9c68f73bd0963e3a3e399f2).

**Review by jeffreyscarpenter — APPROVED** [review]

**Pull request merged**